pkgs/applications/file-managers/nnn/default.nix

```diff
@@ -19,27 +19,27 @@
 assert withIcons -> withNerdIcons == false;
 assert withNerdIcons -> withIcons == false;
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
```

Comment:
why is this being changed?

Reply:
That's new recless style (https://nixos.org/manual/nixpkgs/unstable/#mkderivation-recursive-attributes).

Reply:
Are there any benefits or conventions to use this instead of `rec`?